docs/spec/auth/min_fee.md

```diff
@@ -0,0 +1,15 @@
+# Minimum Fees specifications
+
+Fees serve two purposes for the operator the network.
+
+Fees rate limit the growth of the state in stored by every full node.and the allow for a general purpose for censorship of transactions that consume resources while being of little economic value. Fees are best suited as an antispam mechanism where validators are disinterested in the use of the network and idenities of users
+
+Node operators should set minimum fees and unit costs of gas in each token denomination they wish to support like.
+
+`--minimum=1steak,5photinos --gas_unit_cost=2steak,4photinos`
```

Comment:
On the validator, I believe the config is `--minimum_fees` or they can set it in their `gaiad.toml`. Also, with the implementation of #3044, tokens per unit gas will be a client-side argument (e.g. `--gas=200000 --per-unit-gas=0.00001`). This will be used to calculate the feed if I'm not mistaken.